### PR TITLE
new icon: solidity (original, plain)

### PR DESCRIPTION
**Pull request by maltejur**
imageimage

# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

https://docs.soliditylang.org/en/latest/brand-guide.html#solidity-logo-guidelines

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -3898,6 +3898,22 @@
             }
         ]
     },
+    {
+        "name": "solidity",
+        "tags": [
+            "language"
```
Comment (Panquesito7) [inline devicon.json:3904]:
```suggestion
            "programming",
            "language",
            "blockchain"
```

File: devicon.json [hunk 2]
```diff
@@ -3898,6 +3898,22 @@
...
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#383838"
```
Comment (Thomas-Boi) [inline devicon.json:3938]:
```suggestion
        "color": "#383838",
        "aliases": []
```

**Review by Thomas-Boi — APPROVED** [review]
This looks good to me 👍

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**